Cargo.toml

```diff
@@ -22,15 +22,15 @@ rustls-tls = [
 ]
 
 [dependencies]
-anyhow = "1.0.95"
-clap = { version = "4.5.23", features = ["derive", "string"] }
+anyhow = "1.0.96"
+clap = { version = "4.5.30", features = ["derive", "string"] }
 clap-verbosity-flag = "3.0.2"
 futures = "0.3.31"
-indicatif = "0.17.9"
-rattler = { version = "0.28.4", default-features = false }
+indicatif = "0.17.11"
+rattler = { version = "0.28.7", default-features = false }
 rattler_digest = "1.0.4"
 rattler_conda_types = "0.29.5"
-rattler_index = "0.19.37"
+rattler_index = "0.20.2"
 rattler_lock = "0.22.34"
 rattler_networking = { version = "0.21.8", default-features = false }
```

Comment:
![image](https://github.com/user-attachments/assets/0ab31c77-494c-4bfe-a431-230809749bb5)
not sure why this wasn't updated